### DISCUSSION TITLE
Uma visão geral sobre o ASP.NET Razor Pages - Criando o projeto

### DISCUSSION THREAD

**Discussion by Will082** [balta-io/forum#481] [Tira Dúvidas] [answered]
Olá, tenho uma duvida em relação a criação do projeto, quando inicio ele através da linha de comando com o (dotnet new web -o MyRazorApp) ele gera o projeto realmente "cru" e quando tento criar novas paginas após já ter adicionado o suporte a razor pages em Program.cs somente me aparecem as opções mais simples de criação de arquivos, não sendo possivel adicionar paginas no formato Razor padrão(Index.cshtml, Index.cshtml.cs), diferente de quando inicio um novo projeto diretamente através da interface do Visual Studio, que ai sim me apresenta a opção de criação direta de Razor Pages com toda estrutura, gostaria de saber se esta é uma particularidade da IDE ou realmente pelo fato dele criar o projeto "cru" ficam faltando alguma configurações visto que a criação de ambas as formas geram conteudos bem diferentes ao meu ver.
PS: É uma duvida boba que não atrapalha o processo de aprendizagem visto que posso prosseguir com a criação do projeto normalmente mas como foca sempre na manipulação através do CLI(que curto bastante) achei melhor questionar.

**Comment by andrebaltieri** [comment, marked as answer]
Bom dia, @Will082, como vai?

O comando correto para gerar projetos com Razor pages é:
```
dotnet new razor -o MyRazorApp
```

Ai ele traz a estrutura do Razor!

**Comment by dudinhabranco** [comment]
Eu continuo sem ver nenhum opção de criação de Razor Pages como no minuto 1:01 do vídeo "Criando um Razor Page".
Já testei criar o projeto usando o comando do vídeo e incluindo o suporte.
Já testei criar direto o projeto razor, como sujerem aqui nesta dúvida, na qual o suporte já vêm incluido no program.cs,
mas mesmo assim ainda falta algo que eu não sei o que é.
![image](https://github.com/user-attachments/assets/0c15bf79-2d0f-4c06-b6f3-3c6bd3d407db)

**Reply by andrebaltieri** [reply]
@dudinhabranco só criar utilizando a opção `File` e nomear o arquivo como `NomeDoArquivo.razor` 🚀

**Reply by dudinhabranco** [reply]
Entendi. Muito grato!
